l10n_br_base/res_partner.py

```diff
@@ -124,15 +124,27 @@ def _check_ie(self, cr, uid, ids):
 
         return True
 
+    def _check_ie_duplicated(self, cr, uid, ids):
+        """ Check if the field inscr_est has duplicated value
+        """
+        for partner in self.browse(cr, uid, ids):
+            if not partner.inscr_est \
+                or partner.inscr_est == 'ISENTO':         
```

Comment:
@danimaribeiro 

Acabei de achar ISENTO e ISENTA em um DB que estava importando, creio que se não for numérico o ideal é nem comparar